controllers/booksController.js

```diff
@@ -16,9 +16,28 @@ export const handleNewBook = async (req, res) => {
     const result = await Book.create({
       'title': title,
       'author': author,
+      'readOn': readOn
     });
     res.status(201).json({ 'success': `New book ${title} added!` });
   } catch (err) {
     res.status(500).json({ 'message': err.message });
   }
 };
+
+export const handleGetBooks = async (req, res) => {
+  try {
+    const books = await Book.find().exec();
+    res.status(200).json(books);
+  } catch (err) {
+    res.status(500).json({ 'message': err.message });
+  }
+}
+
+export const getCurrentlyReading = async (req, res) => {
+  try {
+    const books = await Book.find({ currentlyReading: true }).exec();
+    res.status(200).json(books[0]);
```

Comment:
There _should_ only ever be one of these at a time, so for now I am leaving this logic.